app/assets/stylesheets/VenueShowTile.scss

```diff
@@ -37,3 +37,7 @@
     text-align: center;
   }
 }
+
+.rt {
```

Comment:
I would strongly recommend changing this to something more readable.  Unless it's something Skeleton gives you, then ignore me.